### ISSUE TITLE
PRESELECT not parsed correctly

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/OpenedgeAblFormatter#195]
For example,
``` abl
DO PRESELECT EACH Customer:
    FIND NEXT Customer NO-LOCK.
END.
```
causes an error:
![image](https://github.com/user-attachments/assets/4733ada3-3f97-457d-8051-6dd00dba73ef)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/91

**Comment by eglekaz** [comment]
Fixed with version 0.0.45 of tree-sitter-abl
